mongodb-plugins/src/main/java/co/cask/hydrator/plugin/batch/sink/MongoDBBatchSink.java

```diff
@@ -57,7 +59,13 @@ public MongoDBBatchSink(MongoDBSinkConfig config) {
 
   @Override
   public void prepareRun(BatchSinkContext context) throws Exception {
-    context.addOutput(Output.of(config.referenceName, new MongoDBOutputFormatProvider(config)));
+    Configuration conf = new Configuration();
+    String path = conf.get(
+      "mapreduce.task.tmp.dir",
```

Comment:
The mongo sink will append the path with maptaskid/_MONGO_OUT_TEMP, all the temporary mongo data will in this directory.